src/WebJobs.Script/Description/CSharp/PackageManager.cs

```diff
@@ -94,6 +96,26 @@ public static string ResolveNuGetPath()
             return path ?? NuGetFileName;
         }
 
+        internal static string GetNugetPackagesPath()
+        {
+            string nugetHome = null;
+            string home = Environment.GetEnvironmentVariable("HOME");
+            if (!string.IsNullOrEmpty(home))
+            {
+                // We're hosted in Azure
+                // Set the NuGet path to %home%\data\Functions\packages\nuget
+                // (i.e. d:\home\data\Functions\packages\nuget)
+                nugetHome = Path.Combine(home, "data\\Functions\\packages\\nuget");
+            }
+            else
+            {
+                string userProfile = Environment.ExpandEnvironmentVariables("%userprofile%");
+                nugetHome = Path.Combine(userProfile, ".nuget\\packages");
+            }
+
+            return nugetHome;
```

Comment:
Does it need to ensure the folder exists, or does nuget auto-create any missing depth?